modules/images-configuration-blocked-payload.adoc

```diff
@@ -0,0 +1,70 @@
+//Modules included in the following assemblies
```

Comment:
It looks like this file and the updates to image-configuration.adoc were accidentally pulled into your commit/PR. I'm guessing that you accidentally squashed your commit into the commit ahead of yours (which was Kelly's).

There are a few ways that you can resolve this. If you ping me Monday (I'm on ~815am eastern) I'd be happy to walk you through one of the ways to fix it!

Reply:
A few last updates to the diagram - file replaced.